packages/manager/modules/nutanix/src/dashboard/component/install-node-modal/template.html

```diff
@@ -19,6 +19,7 @@
             <oui-field
                 label="{{ 'nutanix_dashboard_install_node_label_ahv_ip' | translate }}"
                 data-error-messages="$ctrl.errorMessages"
+                help-text="{{ $ctrl.installNodeForm.ahvIp.$invalid && $ctrl.installNodeForm.$dirty ? '' : ('nutanix_dashboard_install_ip_tooltip' | translate) }}"
```

Comment:
Why not let the help text displayed ?
```suggestion
                help-text="{{ 'nutanix_dashboard_install_ip_tooltip' | translate }}"
```

Reply:
See with Anta, I don't known. I seen that on the app but may be have the discussion with her together ?